### ISSUE TITLE
Trying to launch a game and getting this error (JSONDecodeError)

### ISSUE THREAD

**Issue by Shazzik** [Garulf/Playnite-Plugin#13]
Trying to launch a game and getting this error

<details open><summary>Trace Log</summary>
<p>

```
Traceback (most recent call last):
  File C:\Users\anser\AppData\Roaming\FlowLauncher\Plugins\Playnite-1.5.3\lib\flox\launcher.py, line 43, in __call__
    results = request_method(*request_parameters) or self._results
  File C:\Users\anser\AppData\Roaming\FlowLauncher\Plugins\Playnite-1.5.3\lib\flox\__init__.py, line 118, in _query
    self.query(query)
  File C:\Users\anser\AppData\Roaming\FlowLauncher\Plugins\Playnite-1.5.3\plugin\main.py, line 98, in query
    self.games = pn.import_games(self.playnite_path)
  File C:\Users\anser\AppData\Roaming\FlowLauncher\Plugins\Playnite-1.5.3\plugin\playnite.py, line 16, in import_games
    data = json.load(f)
  File json\__init__.py, line 293, in load
  File json\__init__.py, line 357, in loads
  File json\decoder.py, line 337, in decode
  File json\decoder.py, line 355, in raw_decode
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

```
</p>
</details>

**Comment by Shazzik** [comment]
Fixed by launching Playnite

**Comment by Garulf** [comment]
I'm glad you were able to get it working. I'll add a better error message for this scenario If possible.

**Comment by Shazzik** [comment]
> I'm glad you were able to get it working. I'll add a better error message for this scenario If possible.

Brilliant! Thank you :D

**Comment by Garulf** [comment]
Should be fixed in https://github.com/Garulf/Playnite-Plugin/pull/23

version 2.0.0

**Comment by Shazzik** [comment]
> Should be fixed in #23
> 
> version 2.0.0

Amazing! Thank you so much 🥰
